src/docs/admin/references/ServiceNowReference.adoc

```diff
@@ -101,461 +101,11 @@ image::servicenow-search-by-status.png[align="center"]
 Result should be empty.
 
 
-==== Communicate with ProActive
+==== Usage
 
-Once the application is installed, you are ready to integrate ProActive in your ServiceNow pipelines.
+Once the application is imported in your ServiceNow instances, you are ready to communication both ways between ProActive and ServiceNow.
```

Comment:
application repository/file/archive/artifact/? 
the two-way communication between ProActive and ServiceNow is ready.

Reply:
Sorry I don't understand the `application repository/file/archive/artifact/?`